src/index.js

```diff
@@ -6,7 +6,10 @@ import { Provider } from "react-redux";
 import rootReducer from "store";
 import App from "App";
 
-const myStore = createStore(rootReducer);
+const myStore = createStore(
+  rootReducer,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
is it required for production build?

Reply:
Added this in order to visualize the data inside redux store through browser, however it is not required for production build. I have tried to figure out if it is necessary/ how to disable it on production build. Found below info in [redux FAQ](https://github.com/reduxjs/redux-devtools/blob/main/extension/docs/FAQ.md). 
[How to disable redux-devtool-extension in production](https://github.com/reduxjs/redux-devtools/blob/main/extension/docs/FAQ.md#how-to-disable-it-in-production)
Thank you vai